src/collapse/collapse.ts

```diff
@@ -0,0 +1,28 @@
+import {Directive, Input} from 'angular2/angular2';
+
+@Directive({
+  selector: '[ngb-collapse]',
```

Comment:
Because we can make everything collapsible without a wrapper element
El 16/11/2015 4:04 p. m., "Pawel Kozlowski" <notifications@github.com>
escribió:

> In src/collapse/collapse.ts
> <https://github.com/ng-bootstrap/core/pull/96#discussion_r44933024>:
>
> > @@ -0,0 +1,28 @@
> > +import {Directive, Input} from 'angular2/angular2';
> > +
> > +@Directive({
> > +  selector: '[ngb-collapse]',
>
> Hmm, I wonder why we would prefer attribute vs. element here. Somehow I
> was assuming:
>
> <ngb-collapse [collapsed]="condition">
>    Content to collapse goes here...
> </ngb-collapse>
>
> but honestly I'm not sure *why* I would lean towards this setup. Thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ng-bootstrap/core/pull/96/files#r44933024>.
>